docs/src/compatibility.md

```diff
@@ -104,3 +104,20 @@ PkgB = "≥ 1.2.3"  # [1.2.3,  ∞)
 PkgC = "= 1.2.3"  # [1.2.3, 1.2.3]
 PkgD = "< 1.2.3"  # [0.0.0, 1.2.2]
 ```
+
+### Hyphen specifiers
+
+Hyphen syntax can also be used to specify version ranges:
+
+```toml
+[compat]
+PkgA = "0.1 - 0.2.3"   # [0.1, 0.2.3]
+```
+
+Any unspecified trailing numbers in the second end-point will be considered to be wildcards:
+
+```toml
+[compat]
+PkgA = "0.1 - 0.2"     # [0.1, 0.3)
```

Comment:
I assume wildcards hold for the first number in the range too? Maybe clarify like:
```suggestion
PkgA = "0.1 - 0.2"     # [0.1.0, 0.3.0)
```

Reply:
> I assume wildcards hold for the first number in the range too? 

I think they just pad with zeros for the first number

Reply:
Yea, thats what I meant, effectively a wildcard though

Reply:
Yeah, sure